tensorflow/tools/pip_package/setup.py

```diff
@@ -39,9 +40,10 @@
     'gast >= 0.2.0',
     'numpy >= 1.13.3',
     'six >= 1.10.0',
-    'protobuf >= 3.4.0',
+    'protobuf >= 3.5.0',
```

Comment:
@martinwicke Is there a particular person in charge of approving PIP dependency changes, and can you please loop them in to this change?

Reply:
If it passes the tests and it's available for all platforms then this is fine.